worker/remoterelations/relationunitsworker.go

```diff
@@ -13,6 +14,13 @@ import (
 	"github.com/juju/juju/apiserver/params"
 )
 
+// RelationUnitChangeEvent encapsulates a remote relation event,
+// adding the tag of the relation which changed.
+type RelationUnitChangeEvent struct {
+	Tag names.RelationTag
+	params.RemoteRelationChangeEvent
```

Comment:
It would be better if the api client extracted the params struct and returned what you actually cared about.

Reply:
Could do but would prefer not to change too much outside of the current behaviour in a PR that is already quite involved.